### PR TITLE
Update DJL Serving multi-node example lws.yaml

### PR DESCRIPTION
**Pull request by xyang16**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: djl-serving/multi-node/lws.yaml [hunk 1]
```diff
@@ -37,18 +37,26 @@ spec:
...
                 value: "$(LWS_LEADER_ADDRESS)"
+              - name: MASTER_ADDR
+                value: "$(LWS_LEADER_ADDRESS)"
               - name: DJL_CLUSTER_SIZE
                 valueFrom:
                   fieldRef:
                     fieldPath: metadata.annotations['leaderworkerset.sigs.k8s.io/size']
               - name: HF_MODEL_ID
-                value: "/opt/ml/model"
+                value: "s3://djl-llm/llama-3.1-405b-hf/"
```
Comment (frankfliu) [inline djl-serving/multi-node/lws.yaml:47]: Use place holder bucket name

File: djl-serving/multi-node/lws.yaml [hunk 2]
```diff
@@ -82,17 +90,23 @@ spec:
...
                 mountPath: /dev/shm
-              - name: model-storage
-                mountPath: /opt/ml/model
               - name: cache-storage
-                mountPath: /tmp/.djl.ai/python/
+                mountPath: /tmp/.djl.ai/
             env:
+              - name: DJL_LEADER_ADDR
+                value: "$(LWS_LEADER_ADDRESS)"
               - name: HF_MODEL_ID
```
Comment (frankfliu) [inline djl-serving/multi-node/lws.yaml:98]: Do we need this on worker node?

File: djl-serving/multi-node/lws.yaml [hunk 3]
```diff
@@ -18,8 +18,8 @@ spec:
             volumeMounts:
               - name: dshm
                 mountPath: /dev/shm
-              - name: model-storage
-                mountPath: /opt/ml/model
+              - name: cache-storage
```
Comment (frankfliu) [inline djl-serving/multi-node/lws.yaml:21]: I think in real world, user still mount to /opt/ml/model, right?

File: djl-serving/multi-node/lws.yaml (hunk 2 above)
Comment (xyang16) [reply djl-serving/multi-node/lws.yaml:98]: Yes, it's needed to report status to leader node.

File: djl-serving/multi-node/lws.yaml (hunk 1 above)
Comment (xyang16) [reply djl-serving/multi-node/lws.yaml:47]: Changed.

File: djl-serving/multi-node/lws.yaml (hunk 3 above)
Comment (xyang16) [reply djl-serving/multi-node/lws.yaml:21]: Added back.

File: djl-serving/multi-node/lws.yaml (hunk 2 above)
Comment (frankfliu) [reply djl-serving/multi-node/lws.yaml:105]: I mean do we need `HF_MODEL_ID` on worker node?

Comment (xyang16) [reply djl-serving/multi-node/lws.yaml:105]: Yes, it is used to download the model.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**